Dockerfile

```diff
@@ -36,17 +36,11 @@ ENV JAVA_HOME=/opt/jdk-mini
 ENV PATH="$PATH:$JAVA_HOME/bin"
```

Comment:
why you need this line if you are starting java process like $JAVA_HOME/bin/java